src/Symfony/Component/Form/ChoiceList/Factory/PropertyAccessDecorator.php

```diff
@@ -86,6 +86,8 @@ public function createListFromChoices($choices, $value = null)
     {
         if (is_string($value) && !is_callable($value)) {
             $value = new PropertyPath($value);
+        } elseif (is_string($value) && is_callable($value)) {
+            @trigger_error('Treating strings as callable is deprecated since version 3.1 and will throw an error in 4.0. You should use a "\Closure" instead.', E_USER_DEPRECATED);
```

Comment:
@xabbuh see my comment above:
>So I suggest to say "Treating callable strings as callable is deprecated since version 3.1 and PropertyAccessDecorator will treat them as strings in 4.0. You should use a "\Closure" instead.".